applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_element.cpp

```diff
@@ -1026,7 +1026,15 @@ void UPwSmallStrainElement<TDim, TNumNodes>::CalculateAll(MatrixType&        rLe
     const auto integration_coefficients =
         this->CalculateIntegrationCoefficients(IntegrationPoints, Variables.detJContainer);
 
+    auto deformation_gradients = std::vector<Matrix>{};
     for (unsigned int GPoint = 0; GPoint < NumGPoints; ++GPoint) {
```

Comment:
Sorry, are GPoint and NumGPoints exceptions from the Kratos style? If not, will they be renamed later?

Reply:
You are right. Those are violations of the Kratos Style Guide. We can rename them, but I would do that in a separate PR, since I didn't touch these variables here (just to separate different concerns). But thanks for pointing this out.